docs.md

```diff
@@ -0,0 +1,1366 @@
+The overwhelming majority of our existing modules are standard modules. 
+
+You should use the **wrapper** template if more than 50% of the content is embedded or linked to external resources (things not hosted in the DART repository). 
+
+You should use the **exercise** template if the module is mostly practice, not instruction. 
+
+## How to use template files
+
+To see how to use a template, you'll need to look at it in its **raw** format.
+
+If you open a template on GitHub, you'll see the rendered GitHub version, which will hide a lot of the important content. 
+To see the raw version on GitHub, click the `Raw` button at the upper right corner of the file. 
+If you open a markdown file in a text editor like VSCode, it will show you the raw file by default.
+
+When you start writing a new module, you'll begin with the template file for the type of module you wish to write, and then fill it in with your content.
+
+To see examples of DART modules, see [our list of completed modules](https://arcus.github.io/education_modules/list_of_modules).
+
+## How to use the front matter
+
+We store a lot of important information in the front matter section at the top of each module. Part of the [module QA process](#module-review-process) is checking that everything is correctly encoded in the front matter. 
+
+The front matter for a module should include all of the elements from the template.
+
+### `author`
+
+```
+author:   Your Name
+```
+
+### `email`
+
+```
+email:    email@example.edu
+```
+
+### `version`
+
+```
+version:  0.0.0
+```
+
+When you're ready to submit your module for QA, it should have a module version number of at least 1.0.0 if first public version, or if this is an update then an [appropriately incremented version number](https://github.com/arcus/education_modules/blob/main/versioning_guidelines.md)
+
+It's a good idea to keep your version set to `0.0.0` while you're still drafting as it will help prevent your browser from caching the module, making it easier for you to review changes as they render online. 
+For more details, see the [LiaScript documentation about state](https://liascript.github.io/course/?https://raw.githubusercontent.com/liaScript/docs/master/README.md#state).
+
+### `module_type`
+
+```
+module_type: standard
+```
+
+Should be one of `standard`, `wrapper`, or `exercise` (see [which module template to use](#which-module-template-to-use)).
+
+### `module_template_version` 
+
+```
+module_template_version: 3.0.0
+```
+
+This value will already be filled in for you when you open the template file. 
+
+When you're ready to submit your module for QA, it should have a template version number that is up to date with the current module template (`template_standard.md`, `template_wrapper.md`, or `template_exercise.md`) -- if not, the module should be brought in line with any changes that have occurred to the module template before continuing submitting it for QA.
+
+### `language`
+
+```
+language: en
+```
```

Comment:
It might be helpful to add an explanatory note here that this is used if a translation of the module is ever created.

Reply:
Good idea, done!